build.sbt

```diff
@@ -571,3 +570,12 @@ lazy val update2_12 = Seq(
     }
   }
 )
+
+lazy val xlint = Seq(
+  scalacOptions += {
+    CrossVersion.partialVersion(scalaVersion.value) match {
+      case Some((2, 12)) => "-Xlint:-unused,_"
```

Comment:
What’s the motivation for this?

I guess in 2.11, `unused` wasn’t part of `lint`, but is there a particular problem with it (asking because I’m in the process of enabling it on all my projects!)